### ISSUE TITLE
[2.8.0-rc] JSX elements no longer suggested imported

### ISSUE THREAD

**Issue by geirsagberg** [Microsoft/TypeScript#22827]
**TypeScript Version:**  2.8.0-rc, 2.9.0-dev.20180323

**Search Terms:** JSX import suggestion

**Code**

```ts
// Foo.tsx
import React from 'react'

export class Foo extends React.Component {
  render () {
    return <div>Hello</div>
  }
}

// Bar.tsx
import React from 'react'

class Bar extends React.Component {
  render () {
    return <Foo /> // Autoimport suggestion expected here
  }
}

```

**Expected behavior:**

![image](https://user-images.githubusercontent.com/594564/37835489-8f9615ba-2eb0-11e8-8fce-43a1f29414f4.png)

**Actual behavior:**

![image](https://user-images.githubusercontent.com/594564/37835546-b2af456c-2eb0-11e8-854d-cbb189620ae3.png)

**Comment by geirsagberg** [comment]
Note I am using 2.7.2 in the top image, it works as expected. I am using 2.8.0-rc in the bottom image, no import suggestion pops up.

**Comment by geirsagberg** [comment]
@andy-ms Thanks for the quick resolve, very impressive :)

Just a question: I see this is tagged as 2.9. Why won't this fix be a part of TS 2.8 RTM? Isn't this the kind of regression bug that RCs are meant to catch and fix before RTM?

**Comment by andy-ms** [comment]
#22901

**Comment by guillaume86** [comment]
Not sure it's the same issue, but since updating to 2.8.1 from 2.7.2 I lost the quick fix for missing import of JSX elements. Typescript output in VS code do not have any errors.

Can you confirm this was included in 2.8?
